### PR TITLE
add kube 1.10.1 and minikube 0.26 to test matrix

### PR DESCRIPTION
**Pull request by csantanapr**

### CODE REVIEW

**Comment by csantanapr** [comment]
@dgrove-oss it looks minikube not coming up in Travis.

I tested locally on OSX and it worked fine for me.

File: docs/setting_up_minikube/README.md [hunk 1]
```diff
@@ -21,21 +21,29 @@ asdf plugin-add kubectl
...
+asdf install kubectl 1.10.1
+asdf global kubectl 1.10.1
 ```
 
 ## Create the minikube VM
 You will want at least 4GB of memory and 2 CPUs for Minikube to run OpenWhisk.
 If you have a larger machine, you may want to provision more (especially more memory).
 
-Start Minikube with:
+Configure minikube and persiste config:
```
Comment (dgrove-oss) [inline docs/setting_up_minikube/README.md]: typo: persist

**Comment by dgrove-oss** [comment]
I kicked it to try again.  Same problem.  minikube 0.26.0 isn't coming up successfully on the travisCI machine.  Probably the next thing to do is to try running it outside of travis on a linux box with the none driver and see what happens.

**Comment by dgrove-oss** [comment]
running setup.sh on an ubuntu 16.04 machine with minikube 0.26.0 / kube 1.10.1 worked for me as well.  Grrrr...hard to fix what you can't reproduce.

**Comment by csantanapr** [comment]
What’s is the vm driver for?

**Comment by dgrove-oss** [comment]
vm-driver=none tells minikube to use the host's docker instead of adding a layer of virtualization.  it's the recommended way to run minikube on travisci to avoid docker-in-docker.

**Comment by csantanapr** [comment]
Maybe we can open an issue on the minikube issue they might be able to help.

**Comment by csantanapr** [comment]
@dgrove-oss you submitted a commit to my branch 💯 😹

**Comment by csantanapr** [comment]
@dgrove-oss Do you know what `minikube update-context` do?
I saw a travis script using this https://github.com/ocadotechnology/aimmo/blob/master/.travis.yml#L26

**Comment by csantanapr** [comment]
@dgrove-oss let's try with ubuntu 16

**Comment by csantanapr** [comment]
ubuntu16 not available on traivs

**Pull request closed without merging**